app/components/candidate_interface/offer_review_component.rb

```diff
@@ -60,5 +77,39 @@ def course_row_value
           tag.p(@course_choice.current_course.description, class: 'govuk-body')
       end
     end
+
+    def salary_row_value
+      if @course_choice.current_course.salary_details
+        markdown(@course_choice.current_course.salary_details)
```

Comment:
I think @duncanjbrown commented on this already. Would it make sense to do the conversion from markdown in `sync_courses.rb` if we have concerns about it slowing the rendering down?

Would need to do the same for `fee_details`

Reply:
Discussed offline